test/FunctionalTests/MvcTests/TagHelperSampleTest.cs

```diff
@@ -47,7 +47,7 @@ public TagHelperSampleTest(SampleTestFixture<TagHelperSample.Web.Startup> fixtur
             AssertRedirectsToHome(createBobbyResponse);
 
             var expectedMediaType = MediaTypeHeaderValue.Parse("text/html; charset=utf-8");
-            var resourceFile = "FunctionalTests.resources.TagHelperSample.Web.Home.Index.html";
+            var resourceFile = "TagHelperSample.Web.Home.Index.html";
```

Comment:
It becomes trickier to figure out the file name when updating (we'd have to remove the dots until `resources.` with slashes). We pass in the resource file name now and construct the full resource location or the path based on what we're doing.